### PR TITLE
Add a check for all unsigned fields when parsing JSON results

### PR DESCRIPTION
**Pull request by tiagobcx**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../docs/code_of_conduct.md). Please see the [contributing guidelines](../docs/contributing.md) for how to create and submit a high-quality PR for this repo.

### Description

- Add a check for all unsigned fields similar to what is done for “column” when parsing JSON.

### References

- https://checkmarx.atlassian.net/browse/AST-16027

### Testing

- Unit and integration tests.

### Checklist

- [x] I have added documentation for new/changed functionality in this PR (if applicable).
- [x] I have updated the CLI help for new/changed functionality in this PR (if applicable).
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

File: internal/wrappers/results-modifier.go [hunk 1]
```diff
@@ -48,19 +50,46 @@ func (s *ScanResult) UnmarshalJSON(data []byte) error {
...
 	if aux.Column >= 0 {
 		s.Column = uint(aux.Column)
+	} else {
+		message = fmt.Sprintf(message, "column", aux.Column, aux.FileName)
+		logger.PrintIfVerbose(message)
+	}
+	if aux.Line >= 0 {
+		s.Line = uint(aux.Line)
+	} else {
+		message = fmt.Sprintf(message, "line", aux.Line, aux.FileName)
```
Comment (diogopcx) [inline internal/wrappers/results-modifier.go:79]: What if column was also negative and message is already reassigned? We should set the format as a constant and do:
```
logger.PrintIfVerbose(fmt.Sprintf(...))
```

We can also define a function that receives an int and contains the if/else statements, returning the value casted to uint or logging and returning 0. Like this we can cut the if/else repetition and do:
```
s.Column = function(aux.Column)
s.Line = ...  
...  
```

Comment (tiagobcx) [reply internal/wrappers/results-modifier.go:79]: Good point, but the message should still be printed in all cases I think. But will check and remove the if repetition with an aux function. Thanks !

Comment (diogopcx) [reply internal/wrappers/results-modifier.go:79]: But in the previous "else" we are doing `message = fmt.Sprintf(message, "line", aux.Line, aux.FileName)` so in the next if/else  the message will not have the %v format placeholders right?

Comment (tiagobcx) [reply internal/wrappers/results-modifier.go:79]: Yes, I was checking and failed. I will change that

**Pull request merged**